1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:09.549558 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:50:14Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:50:16.099270       1 horizontal.go:227] failed to query scale subresource for Deployment/default/dj-app-deployment: deployments/scale.apps \"dj-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:50:16.100445       1 event.go:291] \"Event occurred\" object=\"default/dj-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"dj-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:50:16.500647       1 horizontal.go:227] failed to query scale subresource for Deployment/default/fl-app-deployment: deployments/scale.apps \"fl-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:50:16.500955       1 event.go:291] \"Event occurred\" object=\"default/fl-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"fl-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:19.576762 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:50:23.350084       1 client.go:360] parsed scheme: \"passthrough\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:23.357958 I | embed: rejected connection from \"127.0.0.1:55418\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:50:23.350588       1 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{https://127.0.0.1:2379  <nil> 0 <nil>}] <nil> <nil>}\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:50:23.350701       1 clientconn.go:948] ClientConn switching balancer to \"pick_first\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:23.358645       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:24.363637 I | embed: rejected connection from \"127.0.0.1:55434\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:24.364035       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:25.744060 I | embed: rejected connection from \"127.0.0.1:55442\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:25.744685       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:26.761338       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:27.836815 I | embed: rejected connection from \"127.0.0.1:55460\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:27.837589       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:29.551626 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:32.014421 I | embed: rejected connection from \"127.0.0.1:55494\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:32.015359       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:32.467718       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:39.364462 I | embed: rejected connection from \"127.0.0.1:55556\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:39.364974       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:39.551424 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:50:44Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:44.528273       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:49.552991 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:50:54.335016       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:50:54.335021       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:50:58.537947       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:59.511283 I | mvcc: store.index: compact 1441095\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:59.517835 I | mvcc: finished scheduled compaction at 1441095 (took 934.326Âµs)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:50:59.554418 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:51:02.809585       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:51:02.810114       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:51:02.811159       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/front-proxy-client.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:51:02.812786       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/apiserver.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:51:02.824975       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:51:02.825613       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
